src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,254 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './NewMovie.scss';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: {
+      value: '', isError: true,
+    },
+    imgUrl: {
+      value: '', isError: true,
+    },
+    imdbUrl: {
+      value: '', isError: true,
+    },
+    imdbId: {
+      value: '', isError: true,
+    },
+    description: {
+      value: '', isError: true,
+    },
+  };
+
+  isBtnVisiable = () => {
```

Comment:
Don't use shortened words in names and follow the logic when choosing a name.

```suggestion
  isButtonEnabled = () => {
```